src/services/Store/store/account.slice.ts

```diff
@@ -96,6 +97,29 @@ export const getAccounts = createSelector([getAppState], (s) => {
     }))
   }));
 });
+
+export const selectCurrentAccounts = createSelector(
+  [getAccounts, getFavorites],
+  (accounts, favorites) => {
+    return accounts
+      .filter((a) => a && 'uuid' in a)
```

Comment:
Is it possible to not have a UUID for an account?

Reply:
I don't see how.
Removed.